src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -235,6 +236,15 @@ namespace vcpkg
         VcpkgCmdArguments args;
         std::vector<std::string> feature_flags;
 
+        if (arg_first == arg_last)
+        {
+            args.forwardable_arguments.clear();
```

Comment:
This seems a bit weird, since we just initialized it 6 lines above to empty.

Reply:
Oh I missed that that was touching a local copy with the same type as `*this` rather than `*this` itself.